Sources/JSONRequest.swift

```diff
@@ -102,7 +105,7 @@ open class JSONRequest {
         updateRequest(payload: payload)
 
         let start = Date()
-        let session = networkSession()
+        let session = JSONRequest.urlSession ?? networkSession()
```

Comment:
Where does `JSONRequest.urlSession` get set?  Is it supposed to only be settable by test code?  See earlier comment about documentation